src/librustdoc/html/static/css/rustdoc.css

```
@@ -504,7 +500,11 @@ h2.location a {
 	margin: 0;
 }
 
-.sidebar-links,
+.sidebar-elems .block.items a {
```

I think this would be better:

```suggestion
.sidebar-elems li {
```

The rule doesn't need to be specific to `.block.items`. For instance, it should apply to the "items in this module" section too. Also: selecting on `li` instead of `a` is a nice easy way to say "Headings (h2, h3) can wrap, even when they contain an `<a>` since we generated them and we know they are of a reasonable length. However, list items are Rust item names; they're out of our control, sometimes quite long, and wrap weirdly. Truncate them rather than wrap."